### ISSUE TITLE
Image links for all tutorial are broken

### ISSUE THREAD

**Issue by imoutsatsos** [biouno/biouno.github.io#23]
While viewing tutorials on BioUno all image links seem to be broken 
![untitled_clipping_020518_111337_am](https://user-images.githubusercontent.com/3408226/35815047-9db472b4-0a65-11e8-9663-88514103f100.jpg)

**Comment by imoutsatsos** [comment]
My investigation revealed that the older image, asset (files) and page links use the ```{{ site.baseurl }} ``` annotation. This needs to be replaced with ```{{ site.url }}``` to fix the broken links. I have started fixing a few but we nee to do this site wide.

**Comment by kinow** [comment]
Hummm.. all right. Thought I was missing something in `_config.yml`. Let me batch replace site.baseurl by site.url and see if that fixes it.

**Comment by kinow** [comment]
```
find . -name "*.md" -exec sed -i -e 's|baseurl|url|g' {} \;
find . -name "*.md" -exec sed -i -e 's|url\s*}}\([a-z]\)|url }}/\1|g' {} \;
```

**Comment by kinow** [comment]
Hopefully this, plus an extra commit to fix some edge cases, fixed #23 and #24

**Comment by kinow** [comment]
Thanks!!!
